src/AddTodoForm.jsx

```diff
@@ -1,17 +1,41 @@
-function AddTodoForm(todoprops) {
+import {useState} from 'react';
+
+function AddTodoForm({ onAddTodo, todoList }) {
+    // New state variable and function to handle title change
+    const [todoTitle, setTodoTitle] = useState("");
+
+    function handleTitleChange(event) {
+        console.log(todoTitle);
+        event.preventDefault();
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    }
+
+    // Function to handle form submission
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const todoTitle = event.target.elements.title.value;
-        console.log(todoTitle);
-        todoprops.onAddTodo(todoTitle);
-        event.target.reset();
+        // Calculate nextId based on the current length of todoList
+        const nextId = todoList.length > 0 ? todoList[todoList.length - 1].id + 1 : 1;
```

Comment:
I like your decision about id value. Can you try to use Date.now() according the instruction? 
<img width="550" alt="image" src="https://github.com/user-attachments/assets/9940c70e-f804-4ca1-8691-d0515a0ae9d9">